tensorflow/core/BUILD

```diff
@@ -2115,6 +2116,7 @@ tf_cc_test_mkl(
         "//tensorflow/core/kernels:mkl_pooling_ops",
         "//tensorflow/core/kernels:mkl_relu_op",
         "//tensorflow/core/kernels:mkl_tfconv_op",
+        "//tensorflow/core/kernels:mkl_reshape_op",
```

Comment:
Please move this line up before tfconv_op so that targets are in the alphabetical order. 

1 test failed because of this:
https://ci.tensorflow.org/job/tensorflow-pull-requests-sanity/3948/console